docs/public/secure-registry.adoc

```diff
@@ -0,0 +1,35 @@
+== Introduction to secure registry
+
+=== What is a secure registry?
+
+A secure registry is a registry that user can only use credential to access.
+
+=== Where to host secure registry?
+
+User can host secure registry on private GitHub repository, Enterprise GtiHub repository and cluster.
+
+== Steps for adding secure registry when registry is hosted on GitHub repository
+
+=== Step 1: Creating a new GitHub repository to host the registry
+
+Please follow https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository[this instruction] to create a new GitHub repository and push the registry to the repository. The sample GitHub-hosted registry can be found https://github.com/odo-devfiles/registry/[here].
```

Comment:
> Please follow https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository[this instruction] to create a new GitHub repository and push the registry to the repository.

Please follow https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository[this instruction] to create a new GitHub repository and push the registry to the created repository.

Reply:
Done